src/Css/Color.php

```diff
@@ -296,6 +296,7 @@ static function getArray($color, $alpha = 1.0)
             $c["alpha"] = $alpha;
             $c["hex"] = "cmyk($c[0],$c[1],$c[2],$c[3])";
         } else {
+            $color = preg_replace("/[^0-9A-Fa-f]/", "", $color);
```

Comment:
This looks to be insufficient. This addresses the original error, but introduces a new one by potentially shrinking the size of the hex string so that the `mb_substr` calls result in an "illegal string offset" error.